index.bs

```diff
@@ -8346,7 +8297,7 @@ macros:
 	cc-mode-notes: Has <a>channelCountMode constraints</a>
 	cc-interp: speakers
 	tail-time: Maybe
-	tail-time-notes:  If the {{PannerNode/panningModel}} is set to "{{PanningModelType/HRTF}}", the node will produce non-silent output for silent input due to the inherent processing for head responses. Otherwise there are no tail-time references.
+	tail-time-notes:  If the {{PannerNode/panningModel}} is set to "{{PanningModelType/HRTF}}", the node will produce non-silent output for silent input due to the inherent processing for head responses. Otherwise there are no tail-time.
```

Comment:
Maybe "there are no tail-time" -> ", the tail-time is zero."  But I guess this really depends on how we define tail-time, which we haven't yet.